Makefile.am

```diff
@@ -193,6 +195,9 @@ src_tpm2_quote_SOURCES = src/tpm2_quote.c src/main.c
 src_tpm2_takeownership_SOURCES = src/tpm2_takeownership.c src/main.c
 src_tpm2_getpubek_SOURCES = src/tpm2_getpubek.c src/main.c
 src_tpm2_getpubak_SOURCES = src/tpm2_getpubak.c src/main.c
+src_tpm2_createpolicy_CFLAG = $(AM_CFLAGS) 
```

Comment:
Are this line and the next necessary? None of the other tools require this and I think the values you assign are the default.